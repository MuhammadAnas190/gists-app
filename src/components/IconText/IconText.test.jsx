import { render } from '@testing-library/react';
import { IconText } from './IconText';

describe('IconText Component', () => {
  it('should match snapshot for link', () => {
    const { asFragment } = render(
      <IconText iconName="file" text="File test" link="https://google.com" />
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it('should match snapshot for text', () => {
    const { asFragment } = render(<IconText iconName="file" text="File test" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
