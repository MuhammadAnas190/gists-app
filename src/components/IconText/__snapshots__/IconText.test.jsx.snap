// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconText Component should match snapshot for link 1`] = `
<DocumentFragment>
  <div
    class="sc-aXZVg fxkAHn"
  >
    <span
      class="octicon octicon-file"
    />
    <a
      class="sc-gEvEer dAbXZQ"
      href="https://google.com"
      target="_blank"
    >
      File test
    </a>
  </div>
</DocumentFragment>
`;

exports[`IconText Component should match snapshot for text 1`] = `
<DocumentFragment>
  <div
    class="sc-aXZVg fxkAHn"
  >
    <span
      class="octicon octicon-file"
    />
    <p>
      File test
    </p>
  </div>
</DocumentFragment>
`;
